casanovo/casanovo.py

```diff
@@ -37,6 +37,8 @@
 import torch
 import tqdm
 from lightning.pytorch import seed_everything
+from hashlib import shake_256
+from urllib.parse import urlparse
```

Comment:
**todo:** Same thing, but this should also be moved to the first block of imports because urllib is a standard library.